### DISCUSSION TITLE
Welcome to OMNeT++ Discussions!

### DISCUSSION THREAD

**Discussion by rhornig** [omnetpp/omnetpp#977] [Announcements]
We are opening a discussion board to have a central place where generic questions can be asked and all community members can interact. Please use this area instead of submitting questions as ISSUES.

We also encourage all OMNeT++ users to move their discussions/questions here from the OMNeT++ mailing list. A few advantages using this forum instead of the OMNeT++ list:

- it's much closer to the OMNeT++ codebase, so it's much more discoverable.
- users are directly addressable with @username mentions
- issues, PR-s, releases can be referenced
- better code formatting, syntax highlight, file upload support
- question/answer up/down voting
- editable posts (if you want to amend your original question)
- inline replies and threads
- answers can be marked as 'solved' to help other users identify solved problems
- better moderation support (thread deletion/lockdown/closing)
- labels/categories for the discussions
- discussions can be transferred/converted to/from ISSUES if needed

We hope that you will find this place welcoming and full of useful information.

**Comment by user2038493** [comment]
Good idea!! Is it possible that this feature it not enabled yet? I cannot find the button to start a new discussion

**Reply by rhornig** [reply]
Hmm. I have a big green button on the right side. You don't have that?

<img width="955" alt="image" src="https://github.com/omnetpp/omnetpp/assets/40529/f8b4764f-47b7-42d8-b4f5-e17179f9676b">

Well, obviously, you have to be authenticated (logged in) to see the button.

**Reply by user2038493** [reply]
> Hmm. I have a big green button on the right side. You don't have that?
> <img alt="image" width="955" src="https://user-images.githubusercontent.com/40529/240860880-f8b4764f-47b7-42d8-b4f5-e17179f9676b.png">
> 
> Well, obviously, you have to be authenticated (logged in) to see the button.

Yeah, I was logged in but couldn't see it. Is there anything additional that I need to do? Maybe it's a permission issue

![Captura de pantalla de 2023-05-25 11-19-07](https://github.com/omnetpp/omnetpp/assets/890258/8dd05749-32dd-4845-b45f-002036cdd0c6)

**Reply by rhornig** [reply]
I have no idea why this is not visible. In theory all logged in users that have read access to the repository, can start a discussion: https://docs.github.com/en/discussions/quickstart#creating-a-new-discussion

**Reply by rhornig** [reply]
As you seem to have a new profile on github, there might be some default restrictions on new user accounts.

**Reply by j3po** [reply]
Same issue here - I don't have the button and my account is not new at all...

**Reply by rhornig** [reply]
Thanks for the feedback. I have found the offending settings. New discussion should be visible now.

**Reply by 123smile456** [reply]
> Thanks for the feedback. I have found the offending settings. New discussion should be visible now.

Hello, I would like to ask you a question  
I have created a new project by myself, which has been connected with inet framework, but SceneOsgVisualizer module cannot get background photos, while inet framework project can. Why?
Looking forward to your reply

**Reply by nannuarun** [reply]
Hi,

can anyone help on how to download ANDL plugin for OMNET++. I try to install new software from OMNET++, its installing ANDL IDE by default. But this does not really installing plugin which is required to run andlc command to convert andl script to ned .

**Comment by n-jay** [comment]
Hi @rhornig, are INET-related questions/discussions suited for this or the mailing group?

It would be cool if you guys can open up a discussions tab for the INET-framework [repo](https://github.com/inet-framework/inet) as well if possible.

**Reply by rhornig** [reply]
There is already a [discussion board on the inet](https://github.com/inet-framework/inet/discussions)-repo. Those are pre-created discussions about the showcases and tutorials. 

I was thinking whether to open that discussion board to the public (i.e. anyone would be able to create new topics), but that would severely split the comunity's attention. Right now, the traffic is not **that** heavy, so all OMNeT++ and INET related discussion (and even for different models) are welcomed here.

In short, feel free to discuss INET related issues here. I want to gather as much people here as possible. On longer term we would like to phase out the mailing list as its format is not really suitable for discussions.

**Reply by n-jay** [reply]
> There is already a [discussion board on the inet](https://github.com/inet-framework/inet/discussions)-repo. Those are pre-created discussions about the showcases and tutorials.

My bad, didn't notice that before. 

> I was thinking whether to open that discussion board to the public (i.e. anyone would be able to create new topics), but that would severely split the comunity's attention. 

I agree. Otherwise, its going to end up being a mess with different discussions on disparate places.

> In short, feel free to discuss INET related issues here. I want to gather as much people here as possible. On longer term we would like to phase out the mailing list as its format is not really suitable for discussions.

Gotcha, I also believe that is for the best. 
I too found the mailing list extremely unintuitive. 
GitHub discussions would definitely make it extremely easy to link actual contributions and code and more than anything improve searchability and discovery of issues and past discussions.
